pkgs/development/python-modules/rcfile/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, fetchPypi, buildPythonPackage
+, pythonOlder, isPy3k
+, nose }:
+
+buildPythonPackage rec {
+  pname = "rcfile";
+  version = "0.1.4";
+
+  disabled = pythonOlder "2.6" || (isPy3k && pythonOlder "3.3");
```

Comment:
we only carry python2.7,3.5,3.6,3.7,3.8 and only build the (full) package sets for 2.7 and 3.7. This check should be unnecessary